test/ibmq/runtime/test_runtime_integration.py

```diff
@@ -53,11 +55,12 @@ def prepare_circuits(backend):
 
 def main(backend, user_messenger, **kwargs):
     iterations = kwargs['iterations']
-    interim_results = kwargs.pop('interim_results', {})
-    final_result = kwargs.pop("final_result", {})
+    interim_results = kwargs.pop('interim_results', {{}})
+    final_result = kwargs.pop("final_result", {{}})
```

Comment:
Instead of doing the double brackets format, why not just add another `sleep_per_itr` parameter to the program?

Reply:
Fixed!